### ISSUE TITLE
Add an easy way to install via winget

### ISSUE THREAD

**Issue by atrauzzi** [containers/podman-compose#595]
**Is your feature request related to a problem? Please describe.**
I'm in the process of researching how my team can adopt podman-compose as a docker-compose alternative. In the hopes of keeping the steps to get up and running as simple as possible, it would be lovely to see an installer published to winget.

**Describe the solution you'd like**
Put the installer on winget so that I can do something like `winget install RedHat.Podman-Compose` on Windows.

**Describe alternatives you've considered**
Docker compose 😆

**Comment by septatrix** [comment]
Is there something preventing you from using pip? You will need to have python installed anyhow

**Comment by atrauzzi** [comment]
I don't think it's advisable to rely on that. Windows developers (even other types too I guess)  aren't aiming to get involved in the Python ecosystem to use a tool that isn't focused on a Python-centric task.

A good example of from the past is node gyp and less/sass. It can confound users who are expecting to do one thing but get sucked down a rabbit hole they weren't prepared for.

What makes it worse on Windows is that most of the time, a python installation doesn't necessarily imply that a user has the pip bin directory configured on their path. So that can set off a whole other wild goose chase to figure out "well, I installed it but I still can't run it".

Windows isn't great for distributing software, but for a tool like podman, I think it's wise to just go to the effort to package it all up.  I'm pretty sure there are other python applications that do the same as well -- though I'm not sure how it's accomplished.

**Comment by MeikelLP** [comment]
Very sad answer. "I/We don't like windows so we make it very hard for Windows users to use our tool". Thank you for us having to use Docker.

**Comment by atrauzzi** [comment]
@MeikelLP - I think you misread my comment.

I want an easier way to get it installed.

**Comment by MeikelLP** [comment]
Another alternative would be chocolatey. It's more widely used in the Windows community and pretty easy to setup from what I have seen.

**Comment by Bjego** [comment]
> Another alternative would be chocolatey. It's more widely used in the Windows community and pretty easy to setup from what I have seen.

I don't think that chocolatey is still a valid approach. It was quite okay in the last few years, but now an official solution has been around for quite a while (winget) and you should use the official tools.

And yes - I'd like to see a winget package as well.

**Comment by RihanArfan** [comment]
I'd love to see this too so I can use Podman Compose at my org. It could be shipped as a standalone executable, maybe with https://pyinstaller.org/ to solve the need for Python.

**Comment by atrauzzi** [comment]
I'd rather see it just depend on python if winget can express dependencies.
